macros/generate_model_yaml.sql

```diff
@@ -14,9 +32,7 @@
 {%- set columns = adapter.get_columns_in_relation(relation) -%}
 
 {% for column in columns %}
-    {% do model_yaml.append('      - name: ' ~ column.name | lower ) %}
-    {% do model_yaml.append('        description: "' ~ column_desc_dict.get(column.name | lower,'') ~ '"') %}
-    {% do model_yaml.append('') %}
+    {% set model_yaml = generate_column_yaml(column, model_yaml, column_desc_dict) %}
```

Comment:
After merging with the latest code, the integration tests are yielding an error like:
```
Compilation Error in test test_generate_model_yaml (tests/test_generate_model_yaml.sql)
  'generate_column_yaml' is undefined
```

This should fix that:
```suggestion
    {% set model_yaml = codegen.generate_column_yaml(column, model_yaml, column_desc_dict) %}
```